src/protocol.cpp

```diff
@@ -1544,6 +1559,37 @@ bool CProtocol::EvaluateVersionAndOSMes ( const CVector<uint8_t>& vecData )
     return false; // no error
 }
 
+void CProtocol::CreateRecorderStateMes ( const ERecorderState eRecorderState )
+{
+    CVector<uint8_t> vecData ( 1 ); // 1 byte of data
+    int              iPos = 0;      // init position pointer
+
+    // build data vector
+    // recorder state
+    PutValOnStream ( vecData, iPos, static_cast<uint32_t> ( eRecorderState ), 1 );
+
+    CreateAndSendMessage ( PROTMESSID_RECORDER_STATE, vecData );
+}
+
+bool CProtocol::EvaluateRecorderStateMes(const CVector<uint8_t>& vecData)
+{
+    int iPos = 0; // init position pointer
+
+    // check size
+    if ( vecData.Size() != 2 )
```

Comment:
Why check for size 2? Shouldn't it be 1?

Reply:
Fixed, thanks.